File: VB.NET/SetClipboard/SetClipboard/SetClipboard/Form1.vb
Imports System.Net
Imports System.Text.Json

Public Class Form1


    Private Vui As Cybele.Thinfinity.VirtualUI
    Dim cm As ClipboardMessage
    Private serializedCM As String


    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Vui = New Cybele.Thinfinity.VirtualUI()
        Vui.Start(60)
    End Sub


    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Clipboard.SetText(TextBox1.Text)

        cm = New ClipboardMessage()
        cm.Action = "copy"
        cm.Type = "text/plain"
        cm.Text = TextBox1.Text

        serializedCM = JsonSerializer.Serialize(cm)

        Vui.SendMessage(serializedCM)



    End Sub
End Class

Public Class ClipboardMessage

    Public Property Action() As String
    Public Property Type() As String
    Public Property Text() As String

End Class

File: C#/SetClipboard/SetClipboardCSharp/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using Cybele.Thinfinity;
using System.Text.Json;
using System.Text.Json.Serialization;
using System.Json;



namespace SetClipboardCSharp
{
    public partial class Form1 : Form
    {

        VirtualUI vui;

        public class ClipboardMessage
        {
            public string Action { get; set; }
            public string Type { get; set; }
            public string Text { get; set; }
        }



        public Form1()
        {

            vui = new VirtualUI();
            vui.Start();
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            Clipboard.SetText(textBox1.Text);

            ClipboardMessage cm = new ClipboardMessage();
            cm.Action = "copy";
            cm.Type = "text/plain";
            cm.Text = textBox1.Text;

            string serializedCM = JsonSerializer.Serialize(cm);
            vui.SendMessage(serializedCM);
        }

    }
}
